### PR TITLE
Fix watchlist issues

### PR DESCRIPTION
**Pull request by oriondean**

### CODE REVIEW

File: stockflux-watchlist/src/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Watchlist from './components/watchlist/Watchlist';
+import Watchlist from './components/Watchlist/Watchlist';
```
Comment (AugusteBa) [inline stockflux-watchlist/src/App.js]: Could we keep all folder names lowercase?

File: stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useEffect } from 'react';
 import * as PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { truncate } from '../../services/formatters';
-import Minichart from '../minichart/Minichart';
+import Minichart from '../Minichart/Minichart';
```
Comment (AugusteBa) [inline stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js]: Same here

File: stockflux-watchlist/src/App.js (hunk 1 above)
Comment (oriondean) [reply stockflux-watchlist/src/App.js]: Done

File: stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ const WatchlistCard = props => {
 
   useEffect(() => {
     StockFlux.getMiniChartData(props.symbol).then(response => {
-      if (response.success) {
+      if (response.data) {
```
Comment (AugusteBa) [inline stockflux-watchlist/src/components/watchlist-card/WatchlistCard.js]: I don't understand how it worked for me when I checked for response.success, as it clearly should not have ever worked. Maybe I accidentally did not commit a line or something. Anyway, all works now. :)

**Pull request merged**